dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithEuTaxonomyNonFinancials.cy.ts

```diff
@@ -36,7 +36,13 @@ describe('Component test for the Eu-Taxonomy-Non-Financials view page', () => {
   before(function () {
     cy.fixture('CompanyInformationWithEutaxonomyNonFinancialsPreparedFixtures.json').then(
       (preparedFixtures: FixtureData<EutaxonomyNonFinancialsData>[]) => {
-        fixturesForTests = preparedFixtures;
+        fixturesForTests = preparedFixtures.filter((it) =>
+          [
```

Comment:
why do we only go for full datasets here? Aren't the new fixtures automatically created with valid referenced reports?

Reply:
test relied on the preparedFixtures having three entries and some set fields.